lib/Doctrine/ORM/Event/OnClearEventArgs.php

```diff
@@ -38,7 +38,7 @@ class OnClearEventArgs extends \Doctrine\Common\EventArgs
     private $em;
 
     /**
-     * @var string
+     * @var string|null
```

Comment:
Please review this, I think the string is always true, not nullable.

Reply:
`clear()` can (and should?) be called without an entity name. And from the code it looks like that really happens.